APISpecification.yaml

```diff
@@ -0,0 +1,157 @@
+openapi: 3.0.0
+info:
+  title: Tools api
+  version: 0.0.1
+paths:
+  /api/tools/vep/submissions/{id}:
```

Comment:
Does the name justify the action ? 
`/vep/submissions/{id}` : I would interpret as you want to get some information about the vep job
If the intention is to get the results then something like `/vep/results/{id}` would be more appropriate

Reply:
1) User submits data for a VEP job and receives back a submission id (the missing submit endpoint)
2) User calls the api with the submission id to learn about the status of the submission (the missing status endpoint)
3) Finally, VEP analysis has finished, and user calls the api with the submission id to get the results.

I don't think `/vep/results/{id}` communicates this.

Reply:
I agree with Kamal, because here you're not for expecting a submission instance, but results data (will `submission` or `job` endpoint return results, status, or some other information)? Other alternative could be `submission/{id}/restults`, depending on how will we structure the other URLs: 
- `POST:vep/submit` +` GET:vep/[status|results]/{id}`, or
- `POST:vep/submission` + `GET:vep/submission/{id}/[status|results]`?